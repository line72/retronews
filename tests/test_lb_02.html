<blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</p>
</blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem <code>ipsum</code> dolor sit amet, <code>consectetur adipiscing</code> elit, sed do eiusmod tempor</p>
<pre><code>def hello_world():
  print('hello')
  if True:
    print('world')
  print('!!!')

hello_world()
</code></pre>
