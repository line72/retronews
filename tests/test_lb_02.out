> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
> eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
> elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur
> adipiscing elit, sed do eiusmod tempor

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
elit, sed do eiusmod tempor. Lorem `ipsum` dolor sit amet,
`consectetur adipiscing` elit, sed do eiusmod tempor

| def hello_world():
|   print('hello')
|   if True:
|     print('world')
|   print('!!!')
| 
| hello_world()